templates/central-config.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "xl-deploy.fullname" . }}-central-config
+  labels:
+    chart: {{ template "xl-deploy.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.centralConfiguration.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "xl-deploy.name" . }}
+      component: {{ template "xl-deploy.name" . }}-central-config
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "xl-deploy.name" . }}
+        component: {{ template "xl-deploy.name" . }}-central-config
+        release: {{ .Release.Name }}
+    spec:
+      securityContext:
```

Comment:
This may not work. Have you checked?

Reply:
You mean the securityContext and owner? I deployed in local CRC and everything was working fine. I will check for the group id that is set for the folders

Reply:
the fsGroup is not required

Reply:
Updated based on other pod definitions